package.json

```diff
@@ -4,14 +4,19 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "filter-validate-email": "^1.0.4"
+    "browserslist": "^4.16.6",
+    "elliptic": "^6.5.4",
+    "filter-validate-email": "^1.0.4",
+    "ini": "^1.3.7",
+    "ssri": "^6.0.2",
+    "y18n": "^4.0.1"
```

Comment:
Presumably, all of these _new_ `dependencies` except `filter-validate-email` should be in `devDependencies`, as the only actual production dependency prior to this PR was `filter-validate-email`.
Presumably browserslist, elliptic, ini, ssri, and y18n are devDependencies or peer dependencies of the build tools listed in our own devDependencies.

Reply:
Fair point. Changes made.